cognite/client/_api/vision.py

```diff
@@ -88,13 +89,18 @@ def extract(
         if isinstance(features, VisionFeature):
             features = [features]
 
+        beta_features = [f for f in features if f in VisionFeature.beta_features()]
+        if len(beta_features) > 0:
+            warnings.warn(f"Features {beta_features} are in beta and are still in development")
```

Comment:
Didn't really know what to put in the warning message. Open for suggestions here

Reply:
Sounds good to me